CfC update 2025 (#24)
mlinpl/conference.mlinpl.org: call-for-contributions.md

---
layout: page
title: Call for Contributions (Talks and Posters)
html-title: Call for Contributions</br>(Talks and Posters)
permalink: /call-for-contributions
---

> We are excited to invite you to submit proposals for **contributed talks and posters** for **{{ site.inline-logo }}** and **Student Research Workshop**. The talks and posters will be held at the Main Conference on 16 - 17 October 2025. Student Research Workshop will take place on 15 October 2024, at the start of the conference. We invite you to send your submissions to Call for Contributions (CfC) from 1 May to 31 July 2025.

Reviewer: ```suggestion
We are excited to invite you to submit proposals for **contributed talks and posters** for **{{ site.inline-logo }}** and **Student Research Workshop**. The talks and posters will be held at the Main Conference on 15 - 17 October 2025. Student Research Workshop will take place on 15 October 20245 at the start of the conference. We invite you to send your submissions to Call for Contributions (CfC) from 1 May to 31 July 2025.
```

3. Reinforcement Learning: Bandits, Decision and Control, Exploration, Hierarchical RL, Markov Decision Processes, Multi-Agent RL, Navigation, Planning, etc.
4. Probabilistic Methods: Bayesian methods, Causal Inference, Gaussian Processes, Graphical Models, etc.
5. Applications of Machine Learning: Audio and Speech Processing, Computational Biology, Computer Vision, Natural Language Processing, Robotics, Neuroscience, Physics, Social Good, etc.
6. Social Aspects of Machine Learning: Accountability, AI Ethics, AI Safety, Causality, Biased Data, Fairness, Privacy, Robustness, and Safety, etc.
7. Explainable Machine Learning: Interpretability and Explainability of Machine Learning Models, Methods for Understanding and Explaining Model Decisions, Techniques for Building Transparent and Understandable AI Systems, Human-AI Interaction, etc.
8. Technical Solutions and Machine Learning Resources: Architectures, Competitions, Data Sets and Repositories, Distributed Machine Learning, Implementations, Software Toolkits, etc.
9. Machine Learning Stories: Interesting and insightful competitions, projects, and start-up stories!

> **The talks at the Main Conference should have a length of 20 minutes + 5 minutes for questions. For the poster presentation, we will hold two 1.5-hour sessions during the 2nd and 3rd day of the conference. Student Research Workshop talks should take around 15 minutes + 5 minutes for questions.** To participate in the Call for Contributions, we please provide the title of your talk/poster proposal, an abstract of up to 300 words, your preference towards talk or poster, a short bio, a photo, and a list of your previous talks. Talks and posters should be in English. The submission deadline is 31 July 2025. With the help of the Contributions Committee, we will select the best proposals.

Reviewer: We can leave it as is imo.

The factors that will be considered the most when evaluating submissions include:
- The relevance of the proposed topic in the current machine learning landscape.
- Novelty and uniqueness of the proposed topic. Even if your topic does not fall into any listed areas, do not feel discouraged to submit it! Also, interdisciplinary topics are especially highly encouraged!
- Enjoyable by our audience: a good balance between accessibility and scientific excellence.
- For talk proposals, we will take into account the author’s speaking experience, however, it does not mean that we will reject all inexperienced speakers.
- Talks/posters proposals with clear promotion of the commercial products will be rejected. 

All accepted talks and posters will be presented during the Main Conference (or the Student Research Workshop). 
**The presenting author (the one submitting the application) will receive a free ticket.** 
The co-authors of accepted papers will have a guaranteed option to buy the tickets with the Early Bird prices.

## / Timeline

{% include timeline-inline.html 
    timeline-data=site.data.timeline-cfc
    timeline-col-width=12
    date-col-width=3
    desc-col-width=9
%}

> **The talks and posters accepted at the Main Conference will be presented on 16 - 17 October 2025. The Student Research Workshop will be held on 15 October 2025.**

Reviewer: ```suggestion
**The talks and posters accepted at the Main Conference will be presented on 15 - 17 October 2025. The Student Research Workshop will be held on 15 October 2025.**
```

## / Awards

> The best contributed talks and posters from the Main Conference will be awarded with gift cards. The winners will be decided by the audience votes.
> 

Reviewer: Jeszcze nie wymyśliliśmy tych nagród, nie piszmy może tu "gift cards" xD
```suggestion
The best contributed talks and posters from the Main Conference will be awarded. The winners will be decided by the audience votes.

```

Reviewer 2: +1, w zeszłym roku finalnie nie były to karty podarunkowe

<!-- ## / Submit your talk/poster proposal

<ul class="list-inline banner-social-buttons">
    <li>

>         <a href="https://mlinpl2024cfc.paperform.co/" class="btn btn-default btn-lg"><i class="fa-solid fa-list"></i> Submit your talk/poster proposal</a>

Reviewer: ```suggestion
        <a href="https://mlinpl2025cfc.paperform.co/" class="btn btn-default btn-lg"><i class="fa-solid fa-list"></i> Submit your talk/poster proposal</a>
```
